### ISSUE TITLE
gr plots in jupyter very small

### ISSUE THREAD

**Issue by albi3ro** [JuliaPlots/Plots.jl#1785]
[mozilla.pdf](https://github.com/JuliaPlots/Plots.jl/files/2462951/mozilla.pdf)

As seen in the attached pdf, when I try to plot in gr, using a Jupyter notebook running Julia v0.7 ,
the plot is too tiny for me to work out details.  While this is ok to be small in my test case, I would prefer the plots to be pagewidth or something more readable when I am dealing with a lot more data.  

I may just have something configured wrong, but this started when Julia upgraded.

**Comment by mkborregaard** [comment]
It's a firefox thing, a fix is on it's way

**Comment by albi3ro** [comment]
still tiny in chrome.  What browser works?

**Comment by mkborregaard** [comment]
oh it is? :-O @jheinen knows more

**Comment by jheinen** [comment]
Does [this](https://pgi-jcns.fz-juelich.de/pub/tmp/1785.svg) output work in Chrome? If so, the issue can be closed. Output produced with:
```
using GR
histogram(randn(10000))
savefig("1785.svg")
```
... using GR master.

The "old" output is [here](https://pgi-jcns.fz-juelich.de/pub/tmp/1785bug.svg).

**Comment by KenziTrader** [comment]
In Julia 1.0.1 the plots are normal size. In Julia 0.7 you can set the output to png with `gr(fmt=:png)` to get normal size plots.

**Comment by mikolajpp** [comment]
Unfortunately, Plots are terribly small on 1.0.1 as well #1795

**Comment by jheinen** [comment]
Any combination of Plots >= v0.20.2 and GR >= v0.34.1 should work. Are you using the correct GR run-time (`ENV["GRDIR']=""`; `Pkg.build("GR")` or `] build GR`)?

**Comment by mikolajpp** [comment]
Yes. I have updated julia 1.0.1. The problem exists only with SVG backend, and actually the plot itself is fine, but fonts are always very small. Switching to PNG fixes the issue, but is suboptimal as SVG plots looks much nicer.

**Comment by tamasgal** [comment]
I have the same problem with Julia 1.0.1 in chromium 70.0.3538.77 on Arch Linux.

The plots with the GR backend are really tiny.

**Comment by SirLukeSchande** [comment]
I get the same problem with Julia 1.6.2, Plots v1.15.2, GR v0.49.1, IJulia v1.23.2 and Firefox 90.0.2 as well as Safari Version 14.1.1 on macos 11.4

The div containing the svg figure has `style="width: 150px; height: 100px;"` attributes. This is not the case in `pgfplotsx()`.
Are there any workarounds?

**Comment by jheinen** [comment]
Is there a reason why you can't update `GR.jl` and `Plots.jl` to the most recent versions?

**Comment by SirLukeSchande** [comment]
Thanks for the hint. I don't know why running `]up` did not work. I just updated both to the recent version manually. 
This fixed the problem. Sorry about that. 

And thanks again.
